servicesCore/requestCore_service.py

```diff
@@ -1,35 +1,219 @@
+ import asyncio
+
 from artifacts import *
 from config import config
 from servicesContracts.requestEthereum_service import RequestEthereumService
+import servicesExtensions
+import servicesContracts
 from servicesExtensions.requestSyncrhoneExtensionEscrow_service import RequestSynchroneExtensionEscrowService
 from servicesExternal.ipfs_service import Ipfs
 from servicesExternal.web3_single import Web3Single
 
+EMPTY_BYTES_32 = '0x0000000000000000000000000000000000000000'
 
 class RequestCoreService:
+    """
+    The RequestCoreService class is the interface for the Request Core contract
+    """
+
     def __init__(self):
-        pass
+        self._web3Single = Web3Single.getInstance()
+        self._ipfs = Ipfs.getInstance()
+        # TODO: load requestCoreArtifact using JSONLoader
```

Comment:
Is this still a TODO? It should be loaded automatically by the `artifacts` module.

Reply:
is `artifacts` a module?  I know there is a `JSONLoader` class but I don't see where the `json` files are actually loaded.  In `artifacts/__init__.py` we could just have `requestCoreArtifacts = JSONLoader('requestCoreArtifacts.json')`, same for the other `json` files.

Reply:
no longer at TODO